### DISCUSSION TITLE
充值两年会员无法访问屏蔽网，以前都可以的

### DISCUSSION THREAD

**Discussion by 93624065** [getlantern/lantern-forum-cn#979] [General]
以前都可以正常使用，刚买了两年无论安卓手机端，还是window.桌面都无法访问屏蔽的网站，而且window桌面和手机端总是错乱，他们搞不清谁应该谁应该授权谁？？到现在也解决不了，可以申请退款么？

**Comment by MaxMKMax** [comment]
@93624065 您好. 请问您有下载最新版本吗? 最新版本应该没问题的.

**Comment by 93624065** [comment]
你好，
手机和电脑端都是下载了最新的版本的，还是没有办法访问屏蔽网，
只有一次可以访问，但速度明显比之前买会员的速度差太多了，差不多1分种才能开一个网页
后边就没办法用。另外我自己反复卸载安装了好多次，希望能解决，但都没有效果，而且中间出现手机端和电脑端谁授权谁混乱的问题。
我现在有几个方面感觉请协助看下是不是这些原因造成的：
***@***.******@***.***，登录设备没有变，是不是账号混乱了。
2、登录的软件描述时区不对，我在中国大陆是东8时区。
3、我的手机一直用小米，现在小米出的自己的系统叫小米HYPER OS，这个是不是和安卓一样？

***@***.***


感谢


***@***.***
 
发件人： Mike K
发送时间： 2024-10-22 00:53
收件人： getlantern/lantern-forum-cn
抄送： 93624065; Mention
主题： Re: [getlantern/lantern-forum-cn] 充值两年会员无法访问屏蔽网，以前都可以的 (Discussion #979)
@93624065 您好. 请问您有下载最新版本吗? 最新版本应该没问题的.

**Comment by MaxMKMax** [comment]
@93624065 您好. 我们是无法支持小米系统的. 所以您才会用不了. 非常抱歉啊. 您试试看用window电脑版或别的Android?

**Comment by MaxMKMax** [comment]
@93624065 您提供我支付单据和账号邮箱. 我来让他们帮您解决看看

**Comment by 93624065** [comment]
你好，
***@***.***

谢谢



***@***.***
 
发件人： Mike K
发送时间： 2024-11-19 23:58
收件人： getlantern/lantern-forum-cn
抄送： 93624065; Mention
主题： Re: [getlantern/lantern-forum-cn] 充值两年会员无法访问屏蔽网，以前都可以的 (Discussion #979)
@93624065 您提供我支付单据和账号邮箱. 我来让他们帮您解决看看
